cmd/main.go

```diff
@@ -77,6 +78,12 @@ func main() {
 	prometheus.Unregister(collectors.NewProcessCollector(collectors.ProcessCollectorOpts{}))
 	prometheus.Unregister(collectors.NewGoCollector())
 
+	nodeIP := os.Getenv("NODE_IP")
+	if nodeIP != "" {
+		amqpHost = strings.Replace(amqpHost, "NODE_IP", nodeIP, 1)
```

Comment:
amqpHost is default to `amqp:localhost:5672`. Should the `NODE_IP` be `localhost` here?
```
amqpHost = strings.Replace(amqpHost, "localhost", nodeIP, 1)
```

Reply:
this is  an option to use NODE_IP if required , the user will  set the transport host  value as amqp://NODE_IP:30005 and  the container will read  from env  replace NODE_IP to the actual node ip value 
```
  - name: NODE_IP
              valueFrom:
                fieldRef:
                  fieldPath: status.hostIP
```

Reply:
only replaces the  string NODE_IP with  the ip address

Reply:
got it.